draft-ietf-oauth-attestation-based-client-auth.md

```diff
@@ -279,6 +281,12 @@ Implementers should be aware that using the same client attestation across multi
 
 The guidance provided by {{RFC7519}} and {{RFC8725}} applies.
 
+Replay attack prevention MUST be considered by the protocols using this client authentication method. Possible mechanisms comprise:
+
+- The client uses "jti" (JWT ID) claims for the Client Attestation PoP JWT and the authorization server maintains a list of used (seen) "jti" values for the time of which the JWT would be considered valid based on the applicable "exp" claim. If any Client Attestation PoP JWT would be replayed, the authorization server would recognize the "jti" and respond with an authentication error.
+- The authorization server provides a nonce for the particular transaction and the client uses it for the "nonce" claim in the Client Attestation PoP JWT. The authorization server validates that the nonce matches for the transaction. The authorization server MUST ensure that the nonce provides sufficient entropy.
```

Comment:
may we say something on the minimum length of the nonce?

Reply:
as for all the other parameters, it would be better to give a ref to the RFC that have defined them, for the nonce it would be something like 

shall require the nonce parameter defined in Section 3.1.2.1 of [OIDC](https://openid.net/specs/openid-connect-core-1_0.html) in the authentication request.

Reply:
I don't see OIDC making it explicit about the nonce length either